docs/_articles/en/einstein/einstein-setup.md

```diff
@@ -44,6 +43,11 @@ The Einstein for Developers status icon in the bottom panel of the Visual Studio
 
 You can disable or uninstall the Einstein for Developers extension at any time. Bring up the Extensions view by clicking on the Extensions icon in the Activity Bar on the side of VS Code or the **View: Show Extensions** command (⇧⌘X). Search for "Einstein for Developers" in installed extensions list, and then select **Disable** or **Uninstall**.
 
+### Use Einstein for Developers in Code Builder
+
+Einstein for Developers is disabled by default in new Code Builder environments. Before you enable this extension in a new environment go to **Settings** > **Application** > **Telemetry** and set the dropdown value to `all`, to allow for auditing of the enablement of Salesforce Gen AI functionality, and then enable the extension. Telemetry can be disabled once the extension is enabled. 
```

Comment:
Should we break the explanation or follow up action into a separate sentence or paragraph? Just wondering so it doesn't get lost in the instructions for actually enabling

Reply:
please take a look at the update, @AnanyaJha

Reply:
Looks awesome, thanks so much Sonal!!